pom.xml

```diff
@@ -1307,22 +1311,22 @@
     <jodaTimeVersion>2.1</jodaTimeVersion>
     <jrubyVersion>9.0.4.0</jrubyVersion>
     <jsoupVersion>1.7.2</jsoupVersion>
-    <karafVersion>2.4.3</karafVersion>
-    <karafPaxExamVersion>2.3.9</karafPaxExamVersion>
+    <karafVersion>4.0.8</karafVersion>
     <felixVersion>3.0.0</felixVersion>
     <liquibaseVersion>2.0.5-ONMS20160524b</liquibaseVersion>
     <lmaxDisruptorVersion>3.3.2</lmaxDisruptorVersion>
     <log4jVersion>99.99.99-use-log4j2</log4jVersion>
     <log4j2Version>2.5</log4j2Version>
     <minaVersion>2.0.13</minaVersion>
-    <minionKarafVersion>4.0.5</minionKarafVersion>
+    <minionKarafVersion>${karafVersion}</minionKarafVersion>
```

Comment:
Can we remove the `minionKarafVersion` property now that their are both at parity?